README.md

```diff
@@ -50,15 +49,40 @@ Everytime the image is updated, all tags will be pushed with the latest updates.
 
 Start a cluster:
 
-- ```docker-compose up -d ```
+```bash
+docker-compose up -d
+```
 
 Add more brokers:
 
-- ```docker-compose scale kafka=3```
+```bash
+docker-compose up -d --scale kafka=3
```

Comment:
This introduces an implicit dependency on newer versions of docker-compose.

Reply:
maybe we should use newer versions of docker-compose